apps/api/src/app/integrations/usecases/update-integration/update-integration.usecase.ts

```diff
@@ -83,7 +113,7 @@ export class UpdateIntegration {
 
     if (command.active && ![ChannelTypeEnum.CHAT, ChannelTypeEnum.PUSH].includes(existingIntegration.channel)) {
       await this.deactivateSimilarChannelIntegrations.execute({
-        environmentId: command.environmentId,
+        environmentId,
         organizationId: command.organizationId,
         integrationId: command.integrationId,
         channel: existingIntegration.channel,
```

Comment:
Shouldn't it also be removed(depending on feature falg), as you did in create-integration?